src/components/Item/ItemContent.tsx

```diff
@@ -61,6 +62,21 @@ function useDatePickers(item: Item) {
   }, [boardModifiers, path, item, stateManager]);
 }
 
+function useTagColors(stateManager: StateManager): { [tag: string]: string } {
```

Comment:
This hook is called for every card, which seems to be wasteful since the color settings should ideally just be processed once and passed to every card from a single source. I wasn't really sure what the best approach to that would be in the current architecture. Do you have any thoughts?